src/main/resources/data/wotr/wotr/mob_variant/color_zombie.json

```
@@ -0,0 +1,5 @@
+{
+  "minecraft:max_health": 18.0,
+  "minecraft:attack_damage": 2.5,
+  "minecraft:movement_speed": 0.2
+}
```

Update to the new format:
```suggestion
{
  "stats": {
    "minecraft:max_health": 18.0,
    "minecraft:attack_damage": 2.5,
    "minecraft:movement_speed": 0.2
  },
  "texture": "wotr:textures/entity/mob_variant/zombie/color_zombie.png"
}
```

---

thanks! updated